handlers/analytics.go

```diff
@@ -26,7 +26,7 @@ import (
 
 const (
 	GeoHashPrecision     = 3
-	LogChannelBufferSize = 25000
+	LogChannelBufferSize = 200000
```

Comment:
I hope that some player bug won't cause some enormous kafka events overproduction. Just thinking corner case - all viewers sending 200K events every couple of seconds.

Reply:
yeah, fair point.